docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 
 Orgs
 ^^^^
 
-Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain, making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project. Orgs can also be `shared across multiple projects <TODO: link to persistent orgs section>`_.
+Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain (via a unique identifier of characters), making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project. Orgs can also be shared across multiple projects.
 
-Configuring orgs in CumulusCI is powerful, but comes with some complexity. To review all of the details, read the sections TODO: link "Scratch org environments" and TODO: link persistent org section.
+Configuring orgs in CumulusCI is powerful, but comes with some complexity. To review the details, read the sections [TODO: link Scratch org environments] and [TODO: link persistent org section].
 
 Services
 ^^^^^^^^
 
-Services represent external resources used by CumulusCI automation, such as access to a GitHub account or a MetaDeploy instance. Services are usually, but not always, connected to CumulusCI across projects as part of the global keychain. 
+Services represent external resources used by CumulusCI automation, such as access to a GitHub account or a MetaDeploy instance. Services are usually, but not always, connected to CumulusCI across projects as part of the global keychain. The command ``cci service list`` shows you which services are connected in the context of the current project.
 
-The command ``cci service list`` shows you which services are connected in the context of the current project.
+Global services are easy to use and share and, therefore, we recommend you rely on them for the most part. However, services can also be connected at the project level, which means that they're scoped to a single project and cannot be shared.
 
-Some services can be connected at the project level, which means that they're scoped to a single project and aren't shared. We recommend primarily using global services because they're easier to use, but you may encounter scenarios where, for example, you need to use a specific Dev Hub for one and only one project. Connecting that service with ``cci service connect devhub --project`` supports that use case.
+        Example: If you need to use a specific Dev Hub for one - and only one(!) - project, you can connect to that service with the command ``cci service connect devhub --project``.
```

Comment:
Replace with em dashes.

Reply:
So it's looking like that's not feasible with Sphinx. And one small dash is too short. Can we use two small dashes instead? (Any command with two dashes is in that special ``code`` formatting.)

Reply:
Yep, let's do two small en dashes, then. If we ever convert this content to easyDITA, we can do a quick search and replace.